### ISSUE TITLE
assembly terms are not inferred to be organization terms

### ISSUE THREAD

**Issue by ukemi** [geneontology/go-ontology#13480]
I'm trying to make a figure for a paper and I now notice that the assembly terms are no longer inferred to be subclasses of the corresponding organization terms. They used to be inferred. @cmungall can you have a look and see if you can track this down?

**Comment by ukemi** [comment]
Shouldn't 'results_in_assembly_of' be a sub-property of 'results_in_organization_of?

**Comment by ukemi** [comment]
disassembly too

**Comment by cmungall** [comment]
I'm not sure we ever inferred this.

```
$ curl -L $OBO/go/releases/2016-01-01/go.obo | obo-grep.pl -r 'id: GO:0000045' -

[Term]
id: GO:0000045
name: autophagosome assembly
namespace: biological_process
def: "The formation of a double membrane-bounded structure, the autophagosome, that occurs when a specialized membrane sac, called the isolation membrane, starts to enclose a portion of the cytoplasm." [GOC:autophagy, PMID:9412464]
synonym: "autophagic vacuole assembly" EXACT [GOC:autophagy]
synonym: "autophagic vacuole formation" RELATED [GOC:mah]
synonym: "autophagosome biosynthesis" EXACT []
synonym: "autophagosome formation" EXACT []
synonym: "PAS formation" NARROW []
is_a: GO:0007033 ! vacuole organization
is_a: GO:0070925 ! organelle assembly
relationship: part_of GO:0016236 ! macroautophagy
```

missing is_a to GO:1905037 ! autophagosome organization

**Comment by cmungall** [comment]
Looks like current situation is a mix of asserted and missing

## Missing

```
$ blip-findall -r go_edit "differentium(A,results_in_assembly_of,C),differentium(B,results_in_organization_of,C),\+parent(A,R,B)"  -select A-B -label -no_pred
GO:0000045 ! autophagosome assembly     GO:1905037 ! autophagosome organization
GO:0000147 ! actin cortical patch assembly      GO:0044396 ! actin cortical patch organization
GO:0000921 ! septin ring assembly       GO:0031106 ! septin ring organization
GO:0007416 ! synapse assembly   GO:0050808 ! synapse organization
GO:0030474 ! spindle pole body duplication      GO:0051300 ! spindle pole body organization
GO:0031497 ! chromatin assembly GO:0006325 ! chromatin organization
GO:0036258 ! multivesicular body assembly       GO:0036257 ! multivesicular body organization
GO:0044854 ! plasma membrane raft assembly      GO:0044857 ! plasma membrane raft organization
GO:0060271 ! cilium assembly    GO:0044782 ! cilium organization
GO:0070726 ! cell wall assembly GO:0071555 ! cell wall organization
GO:0070925 ! organelle assembly GO:0006996 ! organelle organization
GO:0071709 ! membrane assembly  GO:0061024 ! membrane organization
GO:0097104 ! postsynaptic membrane assembly     GO:0001941 ! postsynaptic membrane organization
GO:0097105 ! presynaptic membrane assembly      GO:0097090 ! presynaptic membrane organization
GO:0097107 ! postsynaptic density assembly      GO:0097106 ! postsynaptic density organization
GO:0098698 ! postsynaptic specialization assembly       GO:0099084 ! postsynaptic specialization organization
GO:0098785 ! biofilm matrix assembly    GO:0098784 ! biofilm matrix organization
GO:0120031 ! plasma membrane bounded cell projection assembly   GO:0120036 ! plasma membrane bounded cell projection organization
GO:1903232 ! melanosome assembly        GO:0032438 ! melanosome organization
GO:1903863 ! P granule assembly GO:0030719 ! P granule organization
GO:1905877 ! invadopodium assembly      GO:1905863 ! invadopodium organization
```

## Asserted

```
$ blip-findall -r go_edit "differentium(A,results_in_assembly_of,C),differentium(B,results_in_organization_of,C),subclass(A,B)"  -select A-B -label -no_pred
GO:0000915 ! actomyosin contractile ring assembly       GO:0044837 ! actomyosin contractile ring organization
GO:0001765 ! membrane raft assembly     GO:0031579 ! membrane raft organization
GO:0002159 ! desmosome assembly GO:0002934 ! desmosome organization
GO:0006334 ! nucleosome assembly        GO:0034728 ! nucleosome organization
GO:0006461 ! protein complex assembly   GO:0071822 ! protein complex subunit organization
GO:0007043 ! cell-cell junction assembly        GO:0045216 ! cell-cell junction organization
GO:0007130 ! synaptonemal complex assembly      GO:0070193 ! synaptonemal complex organization
GO:0009297 ! pilus assembly     GO:0043711 ! pilus organization
GO:0022607 ! cellular component assembly        GO:0016043 ! cellular component organization
GO:0022618 ! ribonucleoprotein complex assembly GO:0071826 ! ribonucleoprotein complex subunit organization
GO:0030031 ! cell projection assembly   GO:0030030 ! cell projection organization
GO:0030032 ! lamellipodium assembly     GO:0097581 ! lamellipodium organization
GO:0030033 ! microvillus assembly       GO:0032528 ! microvillus organization
GO:0031034 ! myosin filament assembly   GO:0031033 ! myosin filament organization
GO:0031036 ! myosin II filament assembly        GO:0031038 ! myosin II filament organization
GO:0031269 ! pseudopodium assembly      GO:0031268 ! pseudopodium organization
GO:0031507 ! heterochromatin assembly   GO:0070828 ! heterochromatin organization
GO:0032202 ! telomere assembly  GO:0032200 ! telomere organization
GO:0034329 ! cell junction assembly     GO:0034330 ! cell junction organization
GO:0034333 ! adherens junction assembly GO:0034332 ! adherens junction organization
GO:0034377 ! plasma lipoprotein particle assembly       GO:0071827 ! plasma lipoprotein particle organization
GO:0034460 ! uropod assembly    GO:0032796 ! uropod organization
GO:0044780 ! bacterial-type flagellum assembly  GO:0044781 ! bacterial-type flagellum organization
GO:0051017 ! actin filament bundle assembly     GO:0061572 ! actin filament bundle organization
GO:0051225 ! spindle assembly   GO:0007051 ! spindle organization
GO:0051292 ! nuclear pore complex assembly      GO:0006999 ! nuclear pore organization
GO:0051382 ! kinetochore assembly       GO:0051383 ! kinetochore organization
GO:0065003 ! macromolecular complex assembly    GO:0043933 ! macromolecular complex subunit organization
GO:0065004 ! protein-DNA complex assembly       GO:0071824 ! protein-DNA complex subunit organization
GO:0065005 ! protein-lipid complex assembly     GO:0071825 ! protein-lipid complex subunit organization
GO:0065006 ! protein-carbohydrate complex assembly      GO:0071823 ! protein-carbohydrate complex subunit organization
GO:0070649 ! formin-nucleated actin cable assembly      GO:0110009 ! formin-nucleated actin cable organization
GO:0070831 ! basement membrane assembly GO:0071711 ! basement membrane organization
GO:0071668 ! plant-type cell wall assembly      GO:0009664 ! plant-type cell wall organization
GO:0071784 ! endoplasmic reticulum cisternal network assembly   GO:0071783 ! endoplasmic reticulum cisternal network organization
GO:0071787 ! endoplasmic reticulum tubular network formation    GO:0071786 ! endoplasmic reticulum tubular network organization
GO:0071940 ! fungal-type cell wall assembly     GO:0031505 ! fungal-type cell wall organization
GO:0085029 ! extracellular matrix assembly      GO:0030198 ! extracellular matrix organization
GO:0097178 ! ruffle assembly    GO:0031529 ! ruffle organization
```

**Comment by cmungall** [comment]
Your SubPropertyOf makes sense. I can't think why we didn't do this before.

I think the last time I looked in detail at these axioms was when we had a different upper level DP, and the parent term was some disjunctive term like organization and biogenesis, and didn't yield to a simple DP.

In any case we should fix.

Also I note that we are using GOREL IDs in the ontology here. We put RO properties in for this a while ago:

http://purl.obolibrary.org/obo/RO_0002592

The text defs could do with updating here. But simply swapping the xrefs on the props should fix this.

**Comment by dosumis** [comment]
I'm sure this hierarchy was in place before -  I remember adding it and using it for inference.  I switched some cases to the more general genus, but this was very patchy.

**Comment by ukemi** [comment]
I have an old screen shot of the autophagosome term showing the inference, so it was in place at one time.

**Comment by cmungall** [comment]
What seems to have happened is GOREL:0002001 seems to have gotten lost from gorel.owl at some point...

I went ahead and merged #13487, so this should be fixed moving forward. The next inference diff should show the first list of assembly terms above with new correct inferences, plus disassembly too

**Comment by ukemi** [comment]
Just spot checked assembly terms in Protege. This seems to have done the trick because the assembly-organization inferences are back. Thanks @cmungall
